aws/v9.3.4/README.md

```diff
@@ -0,0 +1,45 @@
+# :zap: Giant Swarm Release v9.3.4 for AWS :zap:
+
+This release updates managed apps to the latest releases.
```

Comment:
@sslavic release notes look fine to me from Halo perspective. 

What about the other components? Nothing notable? cc @MartaD for Ludacris components

Reply:
Yeah, nothing notable, no special highlights.

Reply:
I’ve checked with the team and we would like to ask to include fixes for 3 CVEs:
- k8s 1.16.11 fresh from today, not communicated yet
- Flatcar 2512.2.1 - newest stable
- calico 3.10.4 - this aws-operator version is from before CNI so it’s not hard coded anywhere

Reply:
k8s + calico upgrade was included, flatcar requires aws-operator legacy update which is not available yet so will be shipped separately.